example/integration_test/camera_test.dart

```diff
@@ -155,14 +147,8 @@ void main() {
         ScreenBox(
             min: ScreenCoordinate(x: 0, y: 0),
             max: ScreenCoordinate(x: 100, y: 100)));
-    expect(camera.zoom, 10);
     expect(camera.bearing, 20);
     expect(camera.pitch, 30);
-    expect(camera.padding!.top, 1);
-    expect(camera.padding!.left, 2);
-    expect(camera.padding!.bottom, 3);
-    expect(camera.padding!.right, 4);
-    expect(camera.zoom!.round(), 10);
```

Comment:
Why these are not being checked anymore?

Reply:
these camera API won't return a padding, at the moment it seems like only `cameraForCoordinates:camera:coordinatesPadding:maxZoom:offset` will return a padding value.

Reply:
I think in this case returned camera options `padding` should be the same as the `padding` provided by supplied camera options, thus we can re-instantiate padding checks.

Reply:
Additionally, zoom check should also be restored, right?